ql/termstructures/inflation/interpolatedzeroinflationcurve.hpp

```diff
@@ -120,7 +120,7 @@ namespace QuantLib {
           // period
           std::pair<Date,Date> lim =
                 inflationPeriod(yTS->referenceDate() - this->observationLag(), frequency);
-          QL_REQUIRE(lim.first <= dates_[0] && dates_[0] <= lim.second,
+          QL_REQUIRE(lim.first <= dates_[0],
```

Comment:
Why is the second condition no longer necessary? (The message should probably be changed, too.)